portlets/src/main/webapp/vue-app/engagement-center/components/challenges/ChallengeCard.vue

```diff
@@ -99,37 +75,40 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
             </div>
           </div>
         </div>
-        <div class="font-weight-bold tertiary--text text-subtitle-2 mb-auto">
-          {{ challenge && challenge.points }} {{ $t('challenges.label.points') }}
+        <div class="my-auto mx-auto">
+          <v-tooltip bottom>
+            <template #activator="{ on, attrs }">
+              <span
+                class="d-flex-inline position-relative text-truncate-2 font-weight-bold text--secondary" 
+                v-bind="attrs"
+                v-on="on">
+                {{ challengeTitle }}
+              </span>
+            </template>
+            {{ challengeTitle }}
+          </v-tooltip>
         </div>
-        <div :class="isMobile && 'flex-column'" class="d-flex">
-          <div :class="isMobile && 'py-1'" class="d-flex flex-row">
-            <v-icon size="16" class="primary--text ps-1">fas fa-calendar-day</v-icon>
-            <span class="my-auto ms-2" v-sanitized-html="DateInfo"></span>
-          </div>
+        <div :class="isMobile && 'flex-column ms-1'" class="d-flex">
+          <engagement-center-avatars-list
+            :avatars="winnerAvatars"
+            :max-avatars-to-show="3"
+            :avatars-count="announcementCount"
+            :size="27"
+            @open-avatars-drawer="openWinnersDrawer" />
+          <v-spacer />
           <div
-            v-if="noParticipationYet && isActiveChallenge"
-            :class="!isMobile && 'ms-auto'"
-            class="d-flex flex-row">
-            <span
-              class="text-light-color my-auto pe-3 align-self-end text-no-wrap">
-              {{ $t('challenges.label.BeTheFirst') }}
-            </span>
-          </div>
-          <div
-            v-else
-            :class="!isMobile && 'ms-auto'"
+            v-if="!(noParticipationYet && isActiveChallenge)"
+            :class="!isMobile"
             class="winners winnersAvatarsList d-flex flex-nowrap pe-3"
             @click="
               $event.preventDefault();
               $event.stopPropagation();
             ">
-            <engagement-center-avatars-list
-              :avatars="winnerAvatars"
-              :max-avatars-to-show="3"
-              :avatars-count="announcementCount"
-              :size="27"
-              @open-avatars-drawer="openWinnersDrawer" />
+            <v-spacer />
+          </div>
+          <div :class="isMobile && 'py-auto'" class="d-flex flex-row">
```

Comment:
`py-auto` doesn't exist as class in https://github.com/Meeds-io/platform-ui/blob/develop/platform-ui-skin/src/main/webapp/skin/less/vuetify/lib/vuetify.less
Besides, if you mean `my-auto` you may add it this way: `my-auto my-sm-0` if you need to apply an automatic margin in mobile only without having to make it using JS.

```suggestion
          <div class="d-flex flex-row">
```

Reply:
thanks for the info, changes applied !